react/week4/github-user-app/src/components/UserContext.js

```diff
@@ -0,0 +1,60 @@
+import React, { useState, createContext } from "react";
+export const UserContext = createContext();
+
+export const UserProvider = ({ children }) => {
+    const [isLoading, setIsLoading] = useState(false);
+    const [users, setUsers] = useState([]);
+    const [userSearch, setUserSearch] = useState("");
+    const [error, setError] = useState("");
+    const [repos, setRepos] = useState([]);
+
+    const fetchUsers = async (url) => {
```

Comment:
I'd put the `API_URL` somewhere in this file, maybe even this method. 
Right now this method fetches anything you pass in via the `url` param.